From: Stefano Rivera <stefanor@debian.org>
Date: Thu, 8 Oct 2015 10:04:07 -0700
Subject: Remove references to external images from docs

 Fetching resources from remote websites when viewing local documentation could
 breach privacy.
 It's fairly unlikely that anyone viewing this documentation cares about the
 current CI status.
Forwarded: not-needed
---
 README.rst | 17 -----------------
 1 file changed, 17 deletions(-)

diff --git a/README.rst b/README.rst
index bd0b82b..ea92544 100644
--- a/README.rst
+++ b/README.rst
@@ -1,23 +1,6 @@
 Python Object Graphs
 ====================
 
-.. image:: https://travis-ci.org/mgedmin/objgraph.svg?branch=master
-   :target: https://travis-ci.org/mgedmin/objgraph
-   :alt: Build Status
-
-.. image:: https://ci.appveyor.com/api/projects/status/github/mgedmin/objgraph?branch=master&svg=true
-   :target: https://ci.appveyor.com/project/mgedmin/objgraph
-   :alt: Build Status (Windows)
-
-.. image:: https://coveralls.io/repos/mgedmin/objgraph/badge.svg?branch=master
-   :target: https://coveralls.io/r/mgedmin/objgraph?branch=master
-   :alt: Test Coverage
-
-.. image:: https://readthedocs.org/projects/objgraph/badge/?version=latest
-   :target: https://readthedocs.org/projects/objgraph/?badge=latest
-   :alt: Documentation Status
-
-
 ``objgraph`` is a module that lets you visually explore Python object graphs.
 
 You'll need `graphviz <https://www.graphviz.org/>`_ if you want to draw
